### DISCUSSION TITLE
能否使用lerna来管理umi项目

### DISCUSSION THREAD

**Discussion by lebron112** [umijs/umi#6412] [General]
类似这样的目录结构
node_modules
packages
---packaje.json
---project1
------src
------.umirc.ts
---------packaje.json
---------xxx
---------xxx
---project2
------src
------.umirc.ts
---------packaje.json
---------xxx
---------xxx
---project3
------src
------.umirc.ts
---------packaje.json
---------xxx
---------xxx

**Comment by xiaohuoni** [comment]
技术上是可行的，但是我觉得有点意义不明。出了能公用一部分 node_module 之外，好像没有其他意义。除非你这几个 package 之间存在相互依赖，不然我觉得没必要这么做。

**Reply by lebron112** [reply]
我自己尝试了一下，确实如你所言
